### PR TITLE
PIM-7792-7793: get one or several ref entities attributes

### PR DESCRIPTION
**Pull request by NolwennP**
TODO:
- Once PIM-7712-7713 is merged, verify that the reference display is ok.

### CODE REVIEW

File: content/rest-api/resources.md [hunk 1]
```diff
@@ -1127,6 +1127,33 @@ Below is the JSON standard format representing a reference entity.
...
+
+In the Akeneo UI, you can find the attributes that compose a given reference entity by following these simple steps. Go in the `Entities` menu. Click on a reference entity and then, click on the `Attributes` sub menu. Below is an example of the attributes that compose the `Designer` entity in the UI.
+
+![Reference entity attributes](/img/screenshots/v3.0/reference_entity_attribute.png)
+
+Below is the JSON standard format representing a reference entity attribute.
+
+```json
+{
+  "code": "description",
```
Comment (LaurentPetard) [inline content/rest-api/resources.md]: And the labels ?

Comment (NolwennP) [reply content/rest-api/resources.md]: Fixed! Thanks Lolo!

File: content/swagger/resources/reference_entity_attributes/definitions/reference_entity_attribute.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    default: false
+    x-immutable: true
+  scopable:
+    type: boolean
+    description: Whether the attribute is scopable, i.e. can have one value by channel
+    default : false
+    x-immutable: true
+  max_characters:
+    type: integer
+    description: Number maximum of characters allowed for the value of the attribute when the attribute type is `text`
```
Comment (ahocquard) [inline content/swagger/resources/reference_entity_attributes/definitions/reference_entity_attribute.yaml]:
```suggestion
    description: Maximum number of characters allowed for the value of the attribute when the attribute type is `text`
```

File: content/swagger/resources/reference_entity_attributes/routes/reference_entity_attributes_code.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    - "3.0"
+  x-ee: true
+  description: This endpoint allows you to get the information about a given attribute for a given reference entity.
+  parameters:
+    - $ref: '#/parameters/reference_entity_code'
+    - $ref: '#/parameters/code'
+  responses:
+    200:
+      description: OK
+      x-details: Returns the content of the product in JSON standard format
```
Comment (ahocquard) [inline content/swagger/resources/reference_entity_attributes/routes/reference_entity_attributes_code.yaml]:
```suggestion
      x-details: Returns the content of the product in JSON format
```

**Pull request merged**